package.json

```diff
@@ -21,6 +21,7 @@
     "@types/react-dom": "^17.0.14",
     "bootstrap": "^5.1.3",
     "buffer": "^6.0.3",
+    "classnames": "^2.3.1",
```

Comment:
Added this since constructing class name strings is getting messy for some components. It's a widely used and package that's actively maintained. 

Before we usually did something like that:

``` js
className={`${styles['selectableJarContainer']} ${isSelectable ? styles['selectable'] : ''} ${isSelected ? styles['selected'] : ''} `}
```

which is not even fully correct because it will generate trailing whitespaces in the class names string if `isSelectable` and/or `isSelected` is false: `selectableJarContainer  `.

Now we can write the same thing like this:

``` js
className={cx('selectableJarContainer', { selectable: isSelectable, selected: isSelected })}
```

Which is shorter, more readable, and more generates a formally correct class name string for all cases.